spec.html

```diff
@@ -6250,9 +6248,9 @@ <h1>
     <emu-clause id="sec-construct" type="abstract operation">
       <h1>
         Construct (
-          _F_: a function object,
+          _F_: a constructor,
```

Comment:
is this one defined somewhere?

Reply:
Yup: https://tc39.es/ecma262/#constructor

E.g., in the the preamble for [7.3.22 SpeciesConstructor](https://tc39.es/ecma262/#sec-speciesconstructor), "constructor" is autolinked.